srcpkgs/cawbird/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'cawbird'
+pkgname=cawbird
+version=1.2.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config vala glib-devel"
+makedepends="gtk+3-devel json-glib-devel sqlite-devel libsoup-devel gettext-devel vala-devel gst-plugins-base1-devel gst-plugins-bad1-devel gst-libav gspell-devel rest-devel"
+depends="rest gspell gst-plugins-bad1 gst-libav gstreamer1 gst-plugins-good1 sqlite libsoup gettext glib gtk+3 json-glib"
+short_desc="GTK twitter client"
+maintainer="LennyPenny <me@lenny.ninja>"
+license="GPL-3.0-only"
+homepage="https://ibboard.co.uk/cawbird/"
+distfiles="https://github.com/IBBoard/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
Please use `cawbird` rather than `${pkgname}`, no gain while harder to read or copy.